hztb-servicemanager/src/main/java/com/svs/hztb/sm/common/util/HeaderUtil.java

```diff
@@ -55,7 +58,8 @@ public void populateRequestDataFromHeader(RequestData requestData, Map<String, S
 					try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://ec2-52-23-152-124.compute-1.amazonaws.com:9000/coding_rules#rule_key=squid%3AS134)